### PR TITLE
Modify "Blog Resource" to "Xeta"

### PR DESCRIPTION
**Pull request by Xety**
Hello,

I want to modify the name, because now it's not *only* a blog. I have also developed a forum and some others features. ^^

### CODE REVIEW

**Comment by Xety** [comment]
It look like my editor love to remove white space. lol
I need to fix it or it's good ?

**Comment by dereuromark** [comment]
No please revert it.

File: README.md [hunk 1]
```diff
@@ -389,7 +389,7 @@ Based on CakePHP3.x.
 
 ## Demo
 *Web-based (demo) applications and tools.*
-- [Blog Resource](https://github.com/Xety/Xeta) - A resource to help people starting with Cake3.
+- [Xeta](https://github.com/Xety/Xeta) - A resource to help people starting with CakePHP 3.
```
Comment (dereuromark) [inline README.md]: You can drop the 3, as its clear that it is Cake 3

**Comment by jadb** [comment]
Alphabetical order please :)

**Comment by Xety** [comment]
Thanks !

**Pull request merged**